### PR TITLE
refactor: rename runtime to peregrine

### PR DESCRIPTION
**Pull request by weichweich**
## Checklist:

- [ ] I have verified that the code works
  - [ ] No panics! (checked arithmetic ops, no indexing `array[3]` use `get(3)`, ...)
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] This PR does not introduce new custom types
  - [ ] If not, I have opened a companion PR with the type changes in the [KILT types-definitions repository](https://github.com/KILTprotocol/type-definitions/pulls)
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

**Review by Diiaablo95 — APPROVED** [review]
LGTM! Finally, I was wondering when the name chain would happen! 🤌🤌🤌

**Comment by weichweich** [comment]
> LGTM! Finally, I was wondering when the name chain would happen! 🤌🤌🤌

i also thought about rename the standalone to `mashnet-runtime`... 🤔

**Comment by Diiaablo95** [comment]
... and it'll come the time for the repo main as well!

File: nodes/parachain/src/chain_spec/peregrine.rs [hunk 1]
```diff
@@ -39,12 +39,12 @@ use crate::chain_spec::{get_account_id_from_seed, get_from_seed, get_properties,
...
 
 pub fn make_dev_spec(id: ParaId) -> Result<ChainSpec, String> {
-	let properties = get_properties("KILT", 15, 38);
+	let properties = get_properties("PILT", 15, 38);
 	let wasm = WASM_BINARY.ok_or("No WASM")?;
 
 	Ok(ChainSpec::from_genesis(
-		"KILT Local",
-		"kilt_parachain_local_testnet",
+		"KILT Peregrine Local",
```
Comment (wischli) [inline nodes/parachain/src/chain_spec/peregrine.rs:46]: This change has to be applied to the Polkadot Apps in order to connect it with the styling.

File: runtimes/peregrine/Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
 authors = ["KILT <info@kilt.io>"]
 edition = "2018"
-name = "kilt-parachain-runtime"
+name = "peregrine-runtime"
```
Comment (wischli) [inline runtimes/peregrine/Cargo.toml:4]: This change might also effect the Polkadot Apps in regards to the types.

**Review by wischli — APPROVED** [review]
LGTM! But please test with the Polkadot Apps, if the styling and types are still matched properly. I could do this on Monday, if we can postpone the merge until then.

File: nodes/parachain/src/chain_spec/peregrine.rs (hunk 1 above)
Comment (weichweich) [reply nodes/parachain/src/chain_spec/peregrine.rs:46]: This is only for local testing. So it's no problem if we break it

File: runtimes/peregrine/Cargo.toml (hunk 2 above)
Comment (weichweich) [reply runtimes/peregrine/Cargo.toml:4]: just checked. Spiritnet runtime still works

**Pull request merged**